contributing/ansible-development.rst

```diff
@@ -0,0 +1,98 @@
+Ansible roles development
+=========================
+
+.. _Ansible: https://www.ansible.com/
+
+This document describes the conventions and process used by the OME team for
+developing, maintaining and releasing its Ansible_ roles.
+
+The set of rules and procedures described below applies to the official
+OME roles registered in https://github.com/ome/ansible-roles.
+
+Source code
+-----------
+
+The source code of a Java library should be maintained under version control
+using Git_ and hosted on GitHub_ under the
+`openmicroscopy <http://github.com/openmicroscopy/>`__ organization.
+Git repositories should be named as `ansible-role-<ROLE_NAME>`. Each directory
+layout should minimally follow the standard
+`Ansible role layout <https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html#role-directory-structure>`_ including other files and folders for testing and
+deployment. A typical role structure is shown below::
+
+    defaults/           # Default variables
+    handlers/           # Handlers
+    meta/               # Role metadaa
+        main.yml        # Dependencies and Galaxy metadata
+    molecule/           # Test
+    tasks/              # Main list of tasks to be executed
+        main.yml
+    templates/          # Role templates
+    .travis.yml         # CI/deployment configuration file
+    README.md
+
+Versioning
+----------
+
+.. _PEP440: https://www.python.org/dev/peps/pep-0440/#semantic-versioning
+
+Ansible roles must follow the PEP440_ scheme for versioning. Final releases
+must also be compliant with `Semantic Versioning`_ i.e. a final version must
+be expressed as `MAJOR.MINOR.PATCH` where:
+
+- the MAJOR version must be incremented when incompatible API changes are made,
+- the MINOR version must be incremented when functionality is added in a
+  backwards-compatible manner, and
+- the PATCH version must be incremented when you backwards-compatible bug
+  fixes are made.
+
+Final releases must be tagged with a tag matching the version i.e. 
+`MAJOR.MINOR.PATCH` with no prefix for integration with Galaxy.
```

Comment:
It's not quite clear if this is "never any prefix because we might want to use Galaxy" or "you can use prefixes only if you don't give two hoots about Galaxy".